chess2.cpp

```diff
@@ -0,0 +1,882 @@
+#include "chess2.h"
```

Comment:
More code comments are needed to make the code readable

Reply:
- Class definition is not clearly defined. Apart from Chess which represents Board, a Piece abstract class and classes for each unique type of piece would make it far more easy to understand.
- camelCasing has to be followed throughout the codebase
- Chess2 doesn't add any meaning to the name of the class.